gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,124 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+
+    def create_fixed_significance_bin_lc(self, significance, significance_method, energy_range,
+                                            spectrum_extraction, separators=[]):
+        """
+        Create time intervals such that each bin of a light curve reach a given significance
```

Comment:
Add empty line here. Sphinx is super picky about docstring formatting, this will error out in the docs build.

You can do this locally to check (but it's slow):
```
python setup.py build_docs
open docs/_build/html/index.html
```